client-v2/src/components/FrontsEdit/FrontCollectionsOverview.tsx

```diff
@@ -26,13 +36,58 @@ const Container = styled(ContentContainer)`
   width: 380px;
 `;
 
+const CollapseAllButtonContainer = styled('div')`
+  margin-top: 10px;
+  width: 100%;
+  display: flex;
+  justify-content: flex-end;
+  :hover {
+    ${ButtonCircularWithTransition} {
+      background-color: ${({ theme }) =>
+        theme.shared.button.backgroundColorFocused};
+    }
+  }
+`;
+
+const CollapseAllDiv = styled('div')`
+  cursor: pointer;
+  font-family: TS3TextSans-Bold;
+  font-size: 14px;
+`;
+
+const CollapseAllLabel = styled('div')`
+  display: inline-block;
+`;
+
+const CollapseAllButton = ({
+  front,
+  closeAllCollections
+}: CollectionOverviewCollapseAllButtonProps) => (
```

Comment:
I'd be tempted here just to have the `CollapseAll` button be a bit more generic and just take an `onClick` handler (or even `React.HTMLProps<HTMLDivElement>` and just spread them on to the `CollapseAllDiv`).  The consumer can just pass the whole handler in and we still get all of the styles to reuse in another case.

Reply:
Good point. I will refactor.

Reply:
No need to refactor if you don't have time - it was more just an idea. Everything looks really good though!